docs/src/pages/components/view/demo.tsx

```diff
@@ -1,286 +1,78 @@
-            id="color"
-            name="color"
-            value={color}
-            onChange={(event: any) => {
-              setColor(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="customAttribute">
-          <input
-            type="text"
-            placeholder="Set custom attribute"
-            value={customAttribute}
-            onChange={(event: any) => {
-              setCustomAttribute(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="height">
-          <input
-            type="text"
-            placeholder="Set height"
-            value={height}
-            onChange={(event: any) => {
-              setHeight(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet
-          id="isDisabled"
-          order="order-last"
-          direction="flex-row"
-          alignItems="items-center"
-        >
-          <input
-            id="isDisabled"
-            name="disabled"
-            type="checkbox"
-            onChange={() => setDisabled(!disabled)}
-          />
-        </FieldSet>
-
-        <FieldSet id="maxHeight">
-          <input
-            type="text"
-            placeholder="Set maxHeight"
-            value={maxHeight}
-            onChange={(event: any) => {
-              setMaxHeight(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="maxWidth">
-          <input
-            type="text"
-            placeholder="Set maxWidth"
-            value={maxWidth}
-            onChange={(event: any) => {
-              setMaxWidth(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="minHeight">
-          <input
-            type="text"
-            placeholder="Set minHeight"
-            value={minHeight}
-            onChange={(event: any) => {
-              setMinHeight(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="minWidth">
-          <input
-            type="text"
-            placeholder="Set minWidth"
-            value={minWidth}
-            onChange={(event: any) => {
-              setMinWidth(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="role">
-          <input
-            type="text"
-            placeholder="Set role"
-            value={role}
-            onChange={(event: any) => {
-              setAriaRole(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="opacity">
-          <input
-            type="text"
-            placeholder="Set opacity"
-            value={opacity}
-            onChange={(event: any) => {
-              setOpacity(event.target.value);
-            }}
-          />
-        </FieldSet>
+    `<View` +
+    (props.ariaLabel ? `\n  ariaLabel="${props.ariaLabel}"` : '') +
```

Comment:
can we add a simple shared function to do this that we can use across all our docs pages.  might help clean up the docs pages a little bit instead of having these ternaries everywhere.

Reply:
Good idea. I updated PR to include `getPropString` util.